### DISCUSSION TITLE
What is gta banner?

### DISCUSSION THREAD

**Discussion by nothingbutjasur** [YimMenu/YimMenu#963] [Q&A] [answered]
Does this mean that someone reported me?Then it stops the report.Is it really working?
[21:31:29][WARNING/reaction.cpp:24]: Received GTA Banner from xxxxxxx (rid)

**Comment by MarkEcza** [comment, marked as answer]
![image](https://user-images.githubusercontent.com/85032105/217642645-90540b89-4617-4ad8-b981-b53afce01bb3.png)

**Comment by maybegreat48** [comment]
That protection has a lot of false positives. You should just keep it disabled

**Reply by Yimura** [reply]
Related PR #961, there was an issue with us not accepting the event and it would keep getting sent over and over.

**Reply by nothingbutjasur** [reply]
Ok,thx
